config_pod_lifecycle.go

```diff
@@ -31,7 +31,6 @@ func (c *PodLifecycleConfig) Set(ctx context.Context) error {
 func (c *PodLifecycleConfig) ValidateAndDefault() error {
 	catcher := grip.NewSimpleCatcher()
 	if c.MaxParallelPodRequests == 0 {
-		// TODO: (EVG-16217) Determine empirically if this is indeed reasonable
```

Comment:
This ticket is closed -- @Kimchelly you mentioned in a comment that 300 seems reasonable but idk if you meant for this or something else. Should we just leave this default as is?